var/spack/repos/builtin/packages/py-pygelf/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPygelf(PythonPackage):
+    """Python logging handlers with GELF (Graylog Extended Log Format) support."""
```

Comment:
Flake is complaining about this line being too long and the blank line at the end of the file

Reply:
You should remove this file from this PR and we can handle it in your other PR